Tasks/MSBuildV1/package-lock.json

```diff
@@ -321,9 +321,9 @@
       },
       "dependencies": {
         "azure-pipelines-task-lib": {
-          "version": "2.11.3",
-          "resolved": "https://registry.npmjs.org/azure-pipelines-task-lib/-/azure-pipelines-task-lib-2.11.3.tgz",
-          "integrity": "sha512-gXops6Npkloh7AKPIvptx0KY1kdf1yd+BFdp/ksnbyW7CS3uXzBQXU1Dermr7A895TrOd02PI6hmsL8cBlePNA==",
+          "version": "2.12.2",
```

Comment:
Is this expected change? There is no related changes in package.json

Reply:
package-lock.json has been reverted from master branch